workspaces/matomo/.changeset/empty-rabbits-drop.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage-community/plugin-matomo-backend': major
+'@backstage-community/plugin-matomo': major
```

Comment:
Do you really want this to be major?  These will end end up being `1.1.0`. Your call, just want to be sure that's what you want before I approve and merge.

Reply:
Hi @yashoswalyo, with other plugins that we migrated from our plugin repo, we set the version in package.json to the same version as we had before.

1.8.1 for analytics-module-matomo. And then we should go with a minor or patch change here to release 1.8.2 or 1.9.0.

Can you apply these two changes to be aligned with all other migrations we're doing?

Thanks a lot :)

Reply:
Going to hold off to give you a chance to reply to the above @yashoswalyo

Reply:
Ok @christoph-jerolimov I'll update the MR